frontend/src/patientApp/PatientHeader.tsx

```diff
@@ -27,12 +31,23 @@ const PatientHeader = () => {
                   {organization?.name}
                 </span>
                 <span className="prime-organization-name margin-left-0 margin-top-05 text-primary-darker">
-                  COVID-19 Testing Portal
+                  {t("Patient header")}
```

Comment:
An example of the simplest kind of translation. `"Patient header"` is type-checked to make sure it's a key in our translations